proto/src/determined/rbac/v1/rbac.proto

```diff
@@ -167,6 +168,9 @@ enum PermissionType {
 
   // Ability to bind, unbind or overwrite resource pool workspace bindings.
   PERMISSION_TYPE_MODIFY_RP_WORKSPACE_BINDINGS = 10001;
+
+  // Ability to bind, unbind, or overwrite namespace workspace bindings.
+  PERMISSION_TYPE_ADD_WORKSPACE_NAMESPACE_BINDINGS = 11001;
```

Comment:
If this permission allows the user to do more actions than just add, it should be renamed to modify

Reply:
Changed this everywhere (CLI, API, RBAC perms) to `SetWorkspaceNamespaceBinding`, `det w bindings set...`, `PERMISSION_TYPE_SET_WORKSPACE_NAMESPACE_BINDING` per our conversation during weekly sync!